suggester/src/main/java/org/opengrok/suggest/Suggester.java

```diff
@@ -140,23 +141,24 @@ public Suggester(
      */
     public void init(final Collection<NamedIndexDir> luceneIndexes) {
         if (luceneIndexes == null || luceneIndexes.isEmpty()) {
-            logger.log(Level.INFO, "No index directories found, exiting...");
+            LOGGER.log(Level.INFO, "No index directories found, exiting...");
             return;
         }
         if (!projectsEnabled && luceneIndexes.size() > 1) {
             throw new IllegalArgumentException("Projects are not enabled and multiple Lucene indexes were passed");
         }
 
         synchronized (lock) {
-            logger.log(Level.INFO, "Initializing suggester");
+            long startTime = System.currentTimeMillis();
```

Comment:
Why not https://docs.oracle.com/javase/8/docs/api/java/time/Instant.html ?

Reply:
seems too much trouble to convert this to `long` and back.

Reply:
Really? I think `Duration.between(start, Instant.now()).toMillis()` is far more descriptive.

Reply:
all the wrapping of `System` seemed too long winded to me, in reality it's not that bad.